CONTRIBUTING.md

```diff
@@ -61,31 +61,91 @@ You can also clean up generated files:
+
+Note that since the `description` will be included in the CHANGELOG — and may be the only summary of the changes that your colleagues will encounter — it is helpful for you to include a concise summary of _how the package is different_ following the release, written in the [imperative mood](https://chris.beams.io/posts/git-commit/#imperative). For example `fix: Address accessibility bug in the Gizmo component`.
+
+#### Type
 
-For example, a PR title of `feat: change API for X component` with a squashed commit that contains BREAKING CHANGE will create a **major** release, bumping the version from 16.2.1 to 17.0.0.
+The commit `type` is used to indicate the type (i.e. patch, minor, major in [Semantic Versioning](#semantic-versioning)) of change that was included in the release, as well as the context in which that change was made.
 
-For another example, a PR title of `feat: add a new Toggle component` will create a **minor** release, bumping the version from 16.0.0 to 16.1.0.
+There are a number of possible commit types (e.g. `chore`, `docs`, `refactor` etc. — for a complete list, refer to the [Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type) docs), but two have special meaning for the purpose of versioning package releases:
 
-Finally, a PR title of `fix: fix the focus styling on our button component` will create a **patch** release, bumping the version from 16.0.0 to 16.0.1.
+- `fix` — indicates that a change addressed a bug or security concern in the published code, but otherwise had no consequence for the released package's features or API (corresponding to a PATCH version)
 
-## Using your newly released component changes
+- `feat` — indicates that a change added something new to the released package's API or features without affecting existing functionality (corresponding to a MINOR version)
 
-To see your deployed Kaizen Component Library changes in your front-end codebase, you'll need to bump the version of the Kaizen package used. You might edit the `package.json` file to change the package version or run `yarn upgrade --latest @cultureamp/kaizen-component-library`.
+#### Breaking changes
+
+If a pull request includes changes which modify existing behaviour or APIs in a way that is not backwards compatible, that change needs to be marked with a `BREAKING CHANGE: <description>` line (including a description [as above](#description)) somewhere in the commit body of the merged commit to master, in order trigger a MAJOR version update on this release.
+
+Since we are using a squash-and-merge strategy for our pull requests, it is therefore recommended that breaking changes be introduced in their own commits, each with a commit summary in the format `BREAKING CHANGE: <description>`, and any further information written into the commit body.
```

Comment:
```suggestion
Since we are using a squash-and-merge strategy for our pull requests, we recommend that breaking changes be introduced in their own commits, each with a commit summary in the format `BREAKING CHANGE: <description>`, and any further information written into the commit body.
```